Prepare r2.2 (#188)
camaraproject/KnowYourCustomer: CHANGELOG.md

    - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/KnowYourCustomer/r2.2/code/API_definitions/kyc-match.yaml&nocors)

### Added
- Define a phoneNumber pattern in kyc-match yaml file by @GillesInnov35 in https://github.com/camaraproject/KnowYourCustomer/pull/170

### Changed
- Fix example for postalCode and streetNumber by @GillesInnov35 in https://github.com/camaraproject/KnowYourCustomer/pull/169
- KYC-Match: Clarification of "the specification" by @Masa8106 in https://github.com/camaraproject/KnowYourCustomer/pull/165
- Align kyc-match with commonalities for meta-release Spring25 by @fernandopradocabrillo in https://github.com/camaraproject/KnowYourCustomer/pull/176 and https://github.com/camaraproject/KnowYourCustomer/pull/181
- Update test plans according to commonalities guidelines by @fernandopradocabrillo https://github.com/camaraproject/KnowYourCustomer/pull/185
- Remove non-mandatory errors by @fernandopradocabrillo https://github.com/camaraproject/KnowYourCustomer/pull/186

## kyc-fill-in v0.3.0

> **kyc-match v0.3.0 is a release candidate version for the CAMARA KnowYourCustomer Fill-in API**

Reviewer: ```suggestion
**kyc-match v0.3.0 is the public release of the version for the CAMARA KnowYourCustomer Fill-in API**
```

- API definition **with inline documentation**:
    - OpenAPI [YAML spec file](https://github.com/camaraproject/KnowYourCustomer/blob/r2.2/code/API_definitions/kyc-fill-in.yaml)
    - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/KnowYourCustomer/r2.2/code/API_definitions/kyc-fill-in.yaml&nocors)
    - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/KnowYourCustomer/r2.2/code/API_definitions/kyc-fill-in.yaml&nocors)

### Changed
- KYC-Fill-In: Customer:User is a person not an "it" by @AxelNennker in https://github.com/camaraproject/KnowYourCustomer/pull/152
- KYC-Fill-In: Use the same values in the description and example - lowercase by @AxelNennker in https://github.com/camaraproject/KnowYourCustomer/pull/153
- Align kyc-fill-in with Commonalities for meta-release Spring25 by @fernandopradocabrillo in https://github.com/camaraproject/KnowYourCustomer/pull/184 and https://github.com/camaraproject/KnowYourCustomer/pull/181
- Remove non-mandatory errors by @fernandopradocabrillo https://github.com/camaraproject/KnowYourCustomer/pull/186

## kyc-age-verification v0.1.0

> 

Reviewer: Add similar line with "... is the public release of the version ..."
